packages/features/timezone-buddy/components/AvailabilityEditSheet.tsx

```diff
@@ -174,13 +183,39 @@ export function AvailabilityEditSheetForm(props: Props & { data: Data; isPending
           )}
 
           <SheetBody className="mt-4 flex flex-col space-y-4">
+            <div>
+              <Label className="text-emphasis">
+                <>{t("select_availability_to_edit")}</>
+              </Label>
+              <Select
+                value={userAvailabilityOptions.find((option) => option.value === selectedSchedule)}
+                options={userAvailabilityOptions}
+                isDisabled={!userHasDefaultSchedule}
+                onChange={(selected) => {
+                  if (selected) {
+                    const changedSchedule = data.find((schedule) => schedule.id === selected.value);
+                    // should always be present, but added if just for type errors
+                    if (changedSchedule) {
+                      setSelectedSchedule(changedSchedule.id);
+                      form.reset({
+                        ...changedSchedule,
+                        timeZone:
+                          changedSchedule.timeZone || Intl.DateTimeFormat().resolvedOptions().timeZone,
+                        schedule: changedSchedule.availability || [],
+                      });
+                      setWorkingHours(changedSchedule.workingHours);
```

Comment:
For simplicity sake, you can only update on schedule at a time, on changing schedules, any changes made are lost and replaced by the schedule of the newly selected option.